crm_compassion/models/res_partner.py

```diff
@@ -41,7 +41,6 @@ def make_ambassador(self):
             partner = user.partner_id
             user.email = partner.firstname[0].lower() + \
                 partner.lastname.lower() + '@cs.local'
-        portal.action_apply()
```

Comment:
this is not the good fix. You should leave this and search somwhere in the compassion-switzerland repository where we add a 'category_id' to the partner. This is what we want to remove.